src/transformers/file_utils.py

```diff
@@ -234,6 +234,13 @@
 }
 
 
+_is_offline_mode = True if os.environ.get("TRANSFORMERS_OFFLINE", "0") in ["1", "ON", "YES"] else False
```

Comment:
Can we use the truthy value from the `ENV_VARS_TRUE_VALUES = {"1", "ON", "YES", "TRUE"}`? Also ignore case?